bat/RELVER.BAT

```diff
@@ -13,16 +13,49 @@ rem ----- Lvl(%z_NestLevel%) Start %0 %1 %2 %3 %4 %5 %6 %7 %8 %9
 pushd %~dps0..
 set z_Version="Uncontrolled"
 
-git --man-path >NUL
-
+rem -- is git installed?
+git --version >NUL
 set z_Git_Installed=%ERRORLEVEL%
 if %z_Git_Installed% NEQ 0 (echo Git is not installed
                             goto final
-                            ) 
+                            )
 
+rem -- are we in a git repository?
+git rev-parse --is-inside-work-tree >NUL
```

Comment:
Suggestion only. The bash version added 2?&1 on the end of this command. You might do the same. There are a few other places this was done as well.

Reply:
@jyganci  - I am not sure that sysout and syserr redirection works the same way in Windows.. This is a bash construct, not Git. See https://linuxize.com/post/bash-redirect-stderr-stdout/ I am sure that @abekornelis can confirm.

Reply:
That might work - but I am loath to suppress error info that might come in handy in case a problem surfaces. Redirecting the 'standard output' is no problem as that would be - in this case - only useless clutter on the screen.